.github/workflows/brs-commit-checker.yml

```diff
@@ -0,0 +1,93 @@
+name: BRS Commit Checker Report
+on:
+      workflow_dispatch:
+        inputs:
+            fix_version:
+                type: string
+                required: true
+                description: The ICU Jira "Fix Version" semver
+            from_git_ref:
```

Comment:
Do we need this to be so flexible, from git-ref to git-ref?
Working with tags would be a lot easier and more readable
(think from `release-76-1` to `release-77-rc`)

Reply:
This derives from how git works. When you issue a command (ex: `checkout`, `reset`, etc.), what you provide as an argument is called a git ref. If "ref" is an interface/supertype, then concrete types of a ref include tags, commit hashes, and branch names.

As you know, with git, the implementation is the specification. So I presume the flexibility comes from git.

I also personally don't think the the flexibility is a problem in the first place.

Reply:
Does it mean that I can use `release-76-rc`?
And that's a git-ref in GitHub lingo?

In fact in git a tag is basically an alias to a ref, or a pointer to a ref (however you want to look at it)

So maybe it's not really a GitHub lingo, but a git one :-)

Reply:
Yes, `release-76-rc` is a git tag, which is concrete type of a git ref, so it will work. And yes, that's an underlying git thing, not a Github-specific thing.